gravitee-plugin-core/src/main/java/io/gravitee/plugin/core/api/AbstractConfigurablePluginManager.java

```diff
@@ -48,6 +48,13 @@ public String getSchema(String pluginId) throws IOException {
 
     @Override
     public String getSchema(String pluginId, boolean includeNotDeployed) throws IOException {
+        final T plugin = get(pluginId, includeNotDeployed);
+        String schemaProperty = plugin.manifest().properties().get(PluginManifestProperties.SCHEMA_PROPERTY);
+
+        if (schemaProperty != null) {
+            return getFile(pluginId, Paths.get(SCHEMAS_DIRECTORY, schemaProperty), includeNotDeployed);
```

Comment:
When the property is defined, does it mean schemas cannot be put somewhere else than in a subfolder of /schemas?

Looking at the example:
```
documentation=README.adoc
schema=schema-form.json
```

It means that the 'README.adoc' will be taken from 'README.adoc' but 'schema-form.json' will be '**schemas**/schema-form.json'?

Reply:
I'm good with the initial thoughts 👍